netutils/banner.py

```diff
@@ -0,0 +1,33 @@
+"""Functions for working with the banner configuration."""
+import re
+from netutils.constants import CARET_C
+
+
+def delimiter_change(config, from_delimiter, to_delimiter):
+    """Change the banner delimiter.
+
+    Args:
+        config (str): Configuration line containing banner delimiter.
+        from_delimiter (str): Regex pattern to search for the delimiter to replace in the banner.
+        to_delimiter (str): Delimiter to include in the config.
+
+    Returns:
+        str: Configuration with delimiter replaced.
+    """
+    config_line = re.sub(from_delimiter, to_delimiter, config)
+    if to_delimiter == CARET_C:
+        config_line = re.sub(r"\^C+", CARET_C, config_line)
+    return config_line
+
+
+def normalise_delimiter_caret_c(config):
+    """Normalise delimiter to ^C.
+
+    Args:
+        config (str): Configuration line containing banner delimiter.
+
+    Returns:
+        str: Configuration with delimiter normalised to ^C.
+    """
```

Comment:
Example should be included here as well.

Reply:
@jeffkala updated in latest commit